network-store-server/src/test/java/com/powsybl/network/store/server/EmbeddedCassandraFactoryConfig.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.server;
+
+import com.github.nosan.embedded.cassandra.EmbeddedCassandraFactory;
+import com.github.nosan.embedded.cassandra.api.CassandraFactory;
+import com.github.nosan.embedded.cassandra.api.Version;
+import com.github.nosan.embedded.cassandra.artifact.DefaultArtifact;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Scope;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+public class EmbeddedCassandraFactoryConfig {
```

Comment:
It is duplicated with integration module. Maybe we could create a test jar from server module to share it with integration?

Reply:
good idea